backend/src/main/kotlin/be/osoc/team1/backend/security/EmailUtil.kt

```diff
@@ -0,0 +1,65 @@
+package be.osoc.team1.backend.security
+
+import org.springframework.mail.SimpleMailMessage
+import org.springframework.mail.javamail.JavaMailSender
+import org.springframework.mail.javamail.JavaMailSenderImpl
+import java.util.Properties
+import java.util.UUID
+
+/**
+ * This object contains every function needed to make and send emails.
+ */
+object EmailUtil {
+    /**
+     * Set email account to send emails with.
+     */
+    private const val emailAddressSender = "opensummerofcode.info@gmail.com"
+    private const val passwordSender = "nharepxthiwygcpj"
```

Comment:
If you're saving the password here, I suppose it's because you know it's not a security issue. Could you add a comment that explains why this is the case for the others though?

Reply:
@NinjaJay, we discussed this in the last meeting and the others said you knew how to quickly add this password to the environment variables. Could you implement this change?

Reply:
I've already answered in discord but the basic lining is this:

`System.getenv("OSOC_SCHEME") ?: "http"`

the http at the end is the default value to be able to run it locally. Then you can set the env variable if you want something different in deployment or for security measures. You should document on the wiki, that you ahve added a new env variable tho.